jenkinsapi/fingerprint.py

```diff
@@ -56,8 +56,6 @@ def valid(self):
             # extract the status code from it
             response_obj = err.response
             if response_obj.status_code == 404:
-                logging.warn("MD5 cannot be checked if fingerprints are not "
```

Comment:
Yeah, this shouldn't go away. If you find it annoying it could be replaced with a warnings.warn - that would limit the number of times it can display. But definitely don't remove a useful warning.